modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json

```diff
@@ -7,6 +7,28 @@
 		"en_US": "MDF Claim"
 	},
 	"name": "MDFClaim",
+	"objectActions": [
```

Comment:
you can add this one in a object-action directory
[see](https://github.com/liferay/liferay-portal/blob/cfe3d629d6384ff41b18b4bea57a4353bb2763ec/modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/object-actions)

Reply:
Resolved